### ISSUE TITLE
about " vutils.save_image()"

### ISSUE THREAD

**Issue by li-2019** [sg-nm/Operation-wise-attention-network#11]
Hello,thanks for your excellent jobs!

                        # save images
                        vutils.save_image(output.data, './results/Outputs/%05d.png' % (int(i)), padding=0, normalize=False)

My torchvision version may be different from the one above,which will not change the tensor,but my torchivision will .

**Comment by TerryYiDa** [comment]
> Hello,thanks for your excellent jobs! But there is an error in your code :"normalize" should be set "True" .
> 
> ```
>                     # save images
>                     vutils.save_image(output.data, './results/Outputs/%05d.png' % (int(i)), padding=0, normalize=False)
> ```

All the recovered images showed some strange blue distortion with the pre-training model Trained_model/model_best.pth，do you have the same problem?

**Comment by sg-nm** [comment]
Thank you so much for the information. Let me check the result.
